clang/lib/AST/Type.cpp

```diff
@@ -3439,6 +3439,8 @@ StringRef FunctionType::getNameForCallConv(CallingConv CC) {
   case CC_PreserveAll: return "preserve_all";
   case CC_M68kRTD: return "m68k_rtd";
   case CC_PreserveNone: return "preserve_none";
+  case CC_RISCVVectorCall:
```

Comment:
Put this on the same line to match the prevailing style? Add `clang-format off` and `clang-format off` comments around the switch if clang-format gives you trouble.

Reply:
Sure~